test-integration/helpers/ui/user.coffee

```diff
@@ -94,7 +94,7 @@ class User extends TestHelper
     @el.loginSubmit().click()
 
   login: (username, password = 'password') =>
-    @el.loginLink().isDisplayed().then (isDisplayed) =>
```

Comment:
`isDisplayed` checked for immediate display which is needed for times when the `foo` should be there immediately and not have any waits